labs-ios-starter/Model/Apollo/Web Auth/Member.swift

```diff
@@ -6,17 +6,28 @@
 //  Copyright © 2020 Lambda, Inc. All rights reserved.
 //
 
-import Foundation
+import UIKit
 
 
 /// Standard user, leader if topic originator
 struct Member: Equatable, Codable {
 
-    var identifier: UUID?
-    var token: Bearer? //if different from identifier (assign in AuthService)
+    enum CodingKeys: String, CodingKey {
+        case id
+        case email
+        case firstName = "firstname"
+        case lastName = "lastname"
+        case avatarURL = "avatarUrl"
+    }
+
+    var id: String?
     var email: String?
     var firstName: String?
     var lastName: String?
+    var avatarURL: URL?
+
+    //holds image after downloading in UserDetailVC
+    var image: UIImage?
```

Comment:
We also have a method to download images straight into UIImageView as an option. If members were to frequently change avatars, they could either be downloaded live straight into the view, or saved here where this image would want to keep saving a new image when the avatar changes.

Reply:
nice, I noticed that and figured we'd use it - just wanted to get the ball rolling